programs/models.py

```
@@ -757,13 +756,13 @@ class Program(models.Model):
-        Translation,
-        related_name="program_value_type",
-        blank=False,
-        null=False,
-        on_delete=models.PROTECT,
-    )
+    VALUE_TYPE_BENEFIT = "benefit"
+    VALUE_TYPE_TAX_CREDIT = "tax_credit"
+    VALUE_TYPE_CHOICES = [
+        (VALUE_TYPE_BENEFIT, "Benefit"),
+        (VALUE_TYPE_TAX_CREDIT, "Tax Credit"),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)